### PR TITLE
Parser

### PR DESCRIPTION
**Pull request by grafuls**

### CODE REVIEW

**Comment by grafuls** [comment]
https://github.com/grafuls/disruption_generator/issues/8

File: disruption_generator/parser/utils.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import attr
+import six
+
+
+@attr.s(hash=True)
+class Action(object):
+    """
+    A trigger object with the requested disruptive action.
+    """
+    type = attr.ib(validator=attr.validators.instance_of(six.text_type))
```
Comment (ILpinto) [inline disruption_generator/parser/utils.py]: action_type

File: disruption_generator/parsers/experiment_parser.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                timeout = trigger[key]['timeout']
+
+                trigger = Action(name=_name, params=params, target_host=target_host, wait=wait, timeout=timeout)
+                _actions.append(trigger)
+            return _actions
+
+        doc = None
+        try:
+            with open(self.yaml_path, 'r') as f:
+                import ipdb;ipdb.set_trace()
```
Comment (grafuls) [inline disruption_generator/parsers/experiment_parser.py]: remove

File: disruption_generator/parsers/experiment_parser.py [hunk 3]
```diff
@@ -56,21 +59,22 @@ def _init_actions(element_trigger):
...
+                timeout = trigger[key]['timeout']
+
+                trigger = Action(name=_name, params=params, target_host=target_host, wait=wait, timeout=timeout)
+                _actions.append(trigger)
             return _actions
 
         doc = None
         try:
             with open(self.yaml_path, 'r') as f:
+                import ipdb;ipdb.set_trace()
```
Comment (ILpinto) [inline disruption_generator/parsers/experiment_parser.py]: ??

**Pull request merged**